src/main/java/com/ontotext/kafka/GraphDBSinkConfig.java

```diff
@@ -74,10 +76,12 @@ public static TransactionType of(String type) {
 	public static final String AUTH_HEADER_TOKEN_DOC = "GraphDB custom header token";
 
 	public static final String RDF_FORMAT = "graphdb.transaction.rdf.format";
+	public static final String DEFAULT_RDF_TYPE = "ttl";
```

Comment:
What should the default rdf type be? I would have liked to be none but since the validator validates on init the default values it's required